unit_tests/test_base_version.py

```diff
@@ -36,6 +36,24 @@ def test_master(self):
         version_list = general_test(scylla_repo, linux_distro)
         self.assertEqual(version_list, ['5.3'])
 
+    def test_ubuntu_22_azure_5_3(self):
+        scylla_repo = self.url_base + '/branch-5.3/deb/unified/2023-05-23T22:36:08Z/scylladb-5.3/scylla.list'
```

Comment:
shouldn't we use the stable path ?

Reply:
1st, this is only a unittest, and i don't think it will change much from the unittest perspective.. but i can try to change... do we have a "stable path" for 5.3 already?

Reply:
@fgelcer So this is the default repo if `scylla_repo` is not provided? The official repo file is https://s3.amazonaws.com/downloads.scylladb.com/deb/ubuntu/scylla-5.3.list

Reply:
> @fgelcer So this is the default repo if `scylla_repo` is not provided? The official repo file is https://s3.amazonaws.com/downloads.scylladb.com/deb/ubuntu/scylla-5.3.list

no, in this case, this is used only for this unittest... this is a test we run as part of the CI of SCT new code..
it is not used in any SCT jobs... just a matter to verify (in this case) if the `base version` mechanism that calculates based on the repo URL does the job correctly...
i have added a new "feature" to the base version mechanism, as Azure initial support is `5.2` and `2023.1`, so there is no point running upgrade from any older version them these...